kin-recovery/README.md

```diff
@@ -0,0 +1,180 @@
+# Backup and Restore Standalone Module
```

Comment:
I strongly recommend that this `readme.MD` will reside in `kin-website-docs` and we'll link it from here.

Reply:
Already talked with Segev about it and it will be done later

Reply:
:ok_hand: